python/cudf/cudf/core/column/string.py

```diff
@@ -5322,18 +5322,37 @@ def minhash(
         0    [21141582, 403093213, 1258052021]
         1    [962346254, 677440381, 122618762]
         dtype: list
+        >>> seeds = cudf.Series([0, 1, 2], dtype=np.uint64)
+        >>> str_series.str.minhash(seeds, method='MurmurHash3_x64_128')
+        0    [3232308021562742685, 4445611509348165860, 586435843695903598]
+        1    [23008204270530356, 1281229757012344693, 153762819128779913]
+        dtype: list
         """
-        if seeds is None:
-            seeds_column = column.as_column(0, dtype=np.uint32, length=1)
-        elif isinstance(seeds, cudf.Series) and seeds.dtype == np.uint32:
-            seeds_column = seeds._column
-        else:
-            raise ValueError(
-                f"Expecting a Series with dtype uint32, got {type(seeds)}"
+        if method == "MurmurHash3_x86_32":
```

Comment:
Can we rewrite this as a single path for both hash functions, and predefine the type variables (and other parts that change) like this?
```python
seed_type = np.uint32 if method == "MurmurHash3_x86_32" else np.uint64

...

if seeds is None:
    seeds_column = column.as_column(0, dtype=seed_type, length=1)
```

Reply:
Split the code into `minhash()` and `minhash64()` as discussed offline.